drivers/flash/soc_flash_mimxrt.c

```diff
@@ -0,0 +1,268 @@
+/*
+ * Copyright (c) 2019 Riedo Networks Ltd.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ *
+ * This is the driver for the S26KL family of HyperFlash devices connected to i.MX-RT
+ * hybrid micro-controller family. Tested on mimxrt1050_evk. This code is based on
+ * the example "flexspi_hyper_flash_polling_transfer" from NXP's EVKB-IMXRT1050-SDK package.
```

Comment:
Would it work to import the original file into ext/ instead of copy/pasting parts of it here?

Reply:
No. Because the original file is a demo in the form of an application, not library or something reusable as is.